src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/Formatting/TestFiles/Expected/FormatSelection.cshtml

```diff
@@ -12,7 +12,7 @@
             <div class="navbar-collapse collapse">
                 <ul class="nav ">
                     <li>@Html.ActionLink("Home", "Index", "Home")</li>
-                    <li>@Html.ActionLink( "About", "About", "Home")</li>
+                    <li>@Html.ActionLink("About", "About", "Home")</li>
```

Comment:
This was a surprise, but a pretty good change.

Reply:
🥳